Makefile

```diff
@@ -78,7 +78,6 @@ openapi/validate: bin/openapi-generator-cli
 .PHONY: gen/openapi-server
 gen/openapi-server: bin/openapi-generator-cli openapi/validate
 	@if git diff --exit-code --name-only | grep -q "api/openapi/model-registry.yaml" || \
-		git diff --exit-code --name-only | grep -q "api/openapi/model-registry.yaml" || \
```

Comment:
As the pipeline builds the server upon a committed version, the server is never checked.
It seems we have also forgotten to update it some time ago, I wonder if there's a better way to test this?